modules/Microsoft.EventGrid/subscriptions/.test/common/dependencies.bicep

```diff
@@ -0,0 +1,134 @@
+@description('Optional. The location to deploy to.')
+param location string = resourceGroup().location
+
+@description('Required. The name of the Virtual Network to create.')
+param virtualNetworkName string
+
+@description('Required. The name of the Managed Identity to create.')
+param managedIdentityName string
+
+@description('Required. The name of the Event Grid Topic to create.')
+param eventTopicName string
+
+@description('Required. The name of the Service Bus Namespace to create.')
+param serviceBusName string
+
+resource virtualNetwork 'Microsoft.Network/virtualNetworks@2022-01-01' = {
+    name: virtualNetworkName
+    location: location
+    properties: {
+        addressSpace: {
+            addressPrefixes: [
+                '10.0.0.0/24'
+            ]
+        }
+        subnets: [
+            {
+                name: 'defaultSubnet'
+                properties: {
+                    addressPrefix: '10.0.0.0/24'
+                }
+            }
+        ]
+    }
+}
+
+resource privateDNSZone 'Microsoft.Network/privateDnsZones@2020-06-01' = {
+    name: 'privatelink.eventgrid.azure.net'
+    location: 'global'
+
+    resource virtualNetworkLinks 'virtualNetworkLinks@2020-06-01' = {
+        name: '${virtualNetwork.name}-vnetlink'
+        location: 'global'
+        properties: {
+            virtualNetwork: {
+                id: virtualNetwork.id
+            }
+            registrationEnabled: false
+        }
+    }
+}
+
+resource managedIdentity 'Microsoft.ManagedIdentity/userAssignedIdentities@2018-11-30' = {
+    name: managedIdentityName
+    location: location
+}
+
+resource eventTopic 'Microsoft.EventGrid/topics@2022-06-15' = {
+    name: eventTopicName
+    location: location
+    properties:{
+        disableLocalAuth: false
+        publicNetworkAccess: 'Enabled'
+    }
+    identity: {
+        type: 'UserAssigned'
+        userAssignedIdentities: {
+            '${managedIdentity.id}': {}
+        }
+    }
+}
+
+resource serviceBus 'Microsoft.ServiceBus/namespaces@2021-11-01' = {
+    name: serviceBusName
+    location: location
+    properties: {
+        sku: {
+            name: 'Standard'
+            tier: 'Standard'
+            capacity: 1
+        }
+        zoneRedundant: false
+    }
+}
+
+resource serviceBus_Queue 'Microsoft.ServiceBus/namespaces/queues@2021-11-01' = {
```

Comment:
I think so far we rarely used the `_`. I'd suggest to use the simple `queue` or `serviceBusQueue` instead.

Same for the topic below.